frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js

```diff
@@ -42,7 +49,9 @@ class VolumeActionsView extends PureComponent<Props> {
         >
           <RadioButton value={VolumeToolEnum.MOVE}>Move</RadioButton>
           <RadioButton value={VolumeToolEnum.TRACE}>Trace</RadioButton>
```

Comment:
The TRACE tool can also be used to volume annotate. It should be disabled in the same way.